enhancements/network/ovs-dynamic-cpu-pinning.md

```diff
@@ -0,0 +1,203 @@
+---
+title: ovs-dynamic-cpu-pinning
+authors:
+  - "@zeeke"
+reviewers:
+  - "@trozet"
+  - "@MarSik"
+  - "@ricky-rav"
+  - "@cgoncalves"
+approvers:
+  - "@trozet"
+api-approvers:
+  - "None"
+creation-date: 2023-03-31
+last-updated: 2023-03-31
+tracking-link:
+  - https://issues.redhat.com/browse/CNF-5977
+---
+
+# OVS Dynamic CPU Pinning
+
+## Summary
+
+Allow OVS daemons to run on reserved and non-reserved CPUs if the network workload requires
```

Comment:
Will the number of handler/revalidator threads be dynamically resized or a fixed number will be dynamically reaffined ?

Reply:
OVS (>3.0) will dynamically adjust the number of threads according to the affinity.

 https://github.com/openvswitch/ovs/commit/0d23948a598ac609e9865174e0874e782a48d6a8

Reply:
FYI, RHCOS 4.13 includes OVS 3.1.

Reply:
Yes, since >v2.6 the number of handler threads depends on the process affinity. 

https://github.com/openvswitch/ovs/commit/b1e517bd2f818fc7c0cd43ee0b67db4274e6b972

cc @amorenoz